modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig

```diff
@@ -0,0 +1,122 @@
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+  style="min-width: 100%;"
+  class="stylingblock-content-wrapper qs-layout-event-1585">
+  <tbody>
+    <tr>
+      <td style="padding: 0px 15px 15px; " class="stylingblock-content-wrapper">
+        
+        <table cellspacing="0" cellpadding="0" role="presentation" style="width: 100%;">
+          <tbody>
+            <tr>
+              <td valign="top" style="width: 15%; padding-right: 7.5px;">
+
+                <table cellpadding="0" cellspacing="0" width="100%" role="presentation"
+                  style="background-color: #F8F9FA; min-width: 100%;" class="stylingblock-content-wrapper qs-date-badge">
+                  <tbody>
+                    <tr>
+                      <td style="padding: 0px;" class="stylingblock-content-wrapper">
+                        
+                        <!-- QS: EVENT MONTH AND DAY -->
+                        <p style="line-height: 100%; text-align: center;">
+                          <span style="font-family:calibri,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,'Helvetica Neue',Arial,sans-serif; color:#1e5288; font-size:14px; line-height: 0px;">{{ content.az_event_month }}</span><br>
+                          <span style="font-family:calibri,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,'Helvetica Neue',Arial,sans-serif; color:#1e5288; font-size:24px; line-height: 30px;">{{ content.az_event_day }}</span>
+                        </p>
+                        
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+
+              </td>
+              <td valign="top" style="width: 85%; padding-left: 7.5px;">
+
+                <!-- QS: TITLE & LINK (recomended 80 character max) -->
+                <p style="line-height: 130%; margin-top:0;" class="qs-heading">
+                  <a class="qs-link" href="{{ az_canonical_url }}" style="color:#8B0015; font-weight: 700; text-decoration:underline;">
```

Comment:
Look to see if az_canonical_url is applying to events and news

Reply:
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_marketing_cloud/az_marketing_cloud.module#L73

Reply:
Look into if link goes to Trellis event (external) or QS event for Trellis events

Reply:
```
  $content_type = $node->getType();
  if (in_array($content_type, ['az_news', 'az_event'])) {
```